File: src/test/samples/sample.jl
# region FirstRegion
x = 42
# endregion

#region SecondRegion
struct MyClass
  #region InnerRegion
  function method()
    println("Hello")
  end
  #endregion

    #region
    function method2()
      println("Unnamed region")
    end
    #   endregion
end
# endregion

File: src/test/samples/sample.lua
--#region FirstRegion
local x = 42
--#endregion

-- #region SecondRegion  
MyClass = {}
function MyClass:method()
    --    #region   InnerRegion  
  print("Hello")
    --         #endregion    ends InnerRegion  

  --  #region  
    print("Unnamed region")
--#endregion  
end
-- #endregion
